public/image/generate.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * User: river
+ * @link http://proscom.ru
+ * @copyright: Copyright (c) 2016 Простые Коммуникации
+ */
+if (!empty($_POST)) {
+    $url = $_POST['url'];
+    $delay = $_POST['delay'];
+    $size = $_POST['size'];
+    
+    $timestamp = time();
```

Comment:
This is being used as a session key but it only has one second resolution (so if two uploads occur in the same second it will break). Is this ok?